gallery/src/styles/NxList/NxListDefaultExample.html

```diff
@@ -9,7 +9,7 @@
 <h3 class="nx-h3">
   List heading
 </h3>
-<ul class="nx-list">
+<ul class="nx-list" role="list">
```

Comment:
Is this necessary? It's not on the other examples and seems like it'd be redundant

Reply:
> Is this necessary? It's not on some of the other examples and seems like it'd be redundant

It is not. I should have removed it: https://github.com/sonatype/sonatype-react-shared-components/pull/226/commits/8a50d3e2699f67a312bdbaaaaa284d1760d5183e

Reply:
> It is not. I should have removed it: 8a50d3e

It's present is several other examples as well